depends/docker-registry-core/docker_registry/core/boto.py

```diff
@@ -26,14 +26,8 @@
 
 """
 
-import gevent.monkey
-gevent.monkey.patch_all()
```

Comment:
I would not take the chance to remove gevent patching here - though it seems ludicrous, I'm pretty sure there was a reason for it being duplicated here.